examples/storage/cassandra/README.md

```diff
@@ -380,10 +380,10 @@ spec:
   replicas: 4
 ```
 
-The Pet Set will now contain four Pets.
+The StatefulSet will now contain four pods.
```

Comment:
I'm not clear on this either. I understood "pet" to be stateful pod, but it does seem they are not equivalent: From the user guide:

> **Relationship between Pets and Pods**: PetSet requires there be {0..N-1} Pets. Each Pet has a deterministic name - PetSetName-Ordinal, and a unique identity. Each Pet has at most one pod, and each PetSet has at most one Pet with a given identity.

Yet the example shown seems to only contain one pod template, so I'm not sure what a multi-pod pet would look like. Chris, do you have an example that explains this better?